scripts/bootstrap.py

```diff
@@ -30,7 +30,8 @@ def populate_jenkins_config_xml(config_xml, master, name, host, port, role, user
 
     _find_and_set(mesos, './master', master)
     _find_and_set(mesos, './frameworkName', name)
-    _find_and_set(mesos, './jenkinsURL', "http://{}:{}".format(host, port))
+    # This used to be host and port. Switching over to DNS Name to address COPS-3395.
+    _find_and_set(mesos, './jenkinsURL', "http://{}.marathon.autoip.dcos.thisdcos.directory".format(name))
```

Comment:
The Universe config for Jenkins defines two ports and this used to specifically use the nginx port. How is the DNS address selecting the correct port?